### PR TITLE
global: fix Message serialization for celery

### LINKED ISSUE #81: mail: Celery is unable to serialize the Message object
EncodeError: can't serialize <flask_mail.Message object at 0x10cb0f310>

```
File "/Users/eamonnmaguire/.virtualenvs/hepdata3/src/invenio-accounts/invenio_accounts/ext.py", line 63, in delay_security_email
    send_security_email.delay(msg)
  File "/Users/eamonnmaguire/.virtualenvs/hepdata3/lib/python2.7/site-packages/celery/app/task.py", line 453, in delay
    return self.apply_async(args, kwargs)
  File "/Users/eamonnmaguire/.virtualenvs/hepdata3/lib/python2.7/site-packages/celery/app/task.py", line 560, in apply_async
    **dict(self._get_exec_options(), **options)
  File "/Users/eamonnmaguire/.virtualenvs/hepdata3/lib/python2.7/site-packages/celery/app/base.py", line 354, in send_task
    reply_to=reply_to or self.oid, **options
  File "/Users/eamonnmaguire/.virtualenvs/hepdata3/lib/python2.7/site-packages/celery/app/amqp.py", line 305, in publish_task
    **kwargs
  File "/Users/eamonnmaguire/.virtualenvs/hepdata3/lib/python2.7/site-packages/kombu/messaging.py", line 165, in publish
    compression, headers)
  File "/Users/eamonnmaguire/.virtualenvs/hepdata3/lib/python2.7/site-packages/kombu/messaging.py", line 241, in _prepare
    body) = dumps(body, serializer=serializer)
  File "/Users/eamonnmaguire/.virtualenvs/hepdata3/lib/python2.7/site-packages/kombu/serialization.py", line 164, in dumps
    payload = encoder(data)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "/Users/eamonnmaguire/.virtualenvs/hepdata3/lib/python2.7/site-packages/kombu/serialization.py", line 59, in _reraise_errors
    reraise(wrapper, wrapper(exc), sys.exc_info()[2])
  File "/Users/eamonnmaguire/.virtualenvs/hepdata3/lib/python2.7/site-packages/kombu/serialization.py", line 55, in _reraise_errors
    yield
  File "/Users/eamonnmaguire/.virtualenvs/hepdata3/lib/python2.7/site-packages/kombu/serialization.py", line 164, in dumps
    payload = encoder(data)
  File "/Users/eamonnmaguire/.virtualenvs/hepdata3/lib/python2.7/site-packages/msgpack/__init__.py", line 47, in packb
    return Packer(**kwargs).pack(o)
  File "msgpack/_packer.pyx", line 223, in msgpack._packer.Packer.pack (msgpack/_packer.cpp:223)
  File "msgpack/_packer.pyx", line 225, in msgpack._packer.Packer.pack (msgpack/_packer.cpp:225)
  File "msgpack/_packer.pyx", line 184, in msgpack._packer.Packer._pack (msgpack/_packer.cpp:184)
  File "msgpack/_packer.pyx", line 213, in msgpack._packer.Packer._pack (msgpack/_packer.cpp:213)
  File "msgpack/_packer.pyx", line 220, in msgpack._packer.Packer._pack (msgpack/_packer.cpp:220)
EncodeError: can't serialize <flask_mail.Message object at 0x10cb0f310>
```

### PR DESCRIPTION
**Pull request by jirikuncar**
* Fixes default serializer of task sending Flask-Mail messages.
  (closes #81)

Signed-off-by: Jiri Kuncar <jiri.kuncar@cern.ch>

### CODE REVIEW

**Comment by eamonnmag** [comment]
:shipit:

**Comment by lnielsen** [comment]
See comment on #85 related to pickle security and deprecation.

File: invenio_accounts/tasks.py [hunk 1]
```diff
@@ -28,9 +28,12 @@
...
 from flask import current_app
+from flask_mail import Message
 
 
 @shared_task
-def send_security_email(msg):
+def send_security_email(data):
     """Celery task to send security email."""
+    msg = Message()
+    msg.__dict__.update(data)
```
Comment (jirikuncar) [inline invenio_accounts/tasks.py]: @lnielsen @eamonnmag please check following solution to avoid security issues with pickle serialization. Thanks

**Comment by jirikuncar** [comment]
ping @lnielsen

**Pull request merged**